libs/repositories/src/UsdRepository.ts

```diff
@@ -0,0 +1,16 @@
+import { get } from 'http';
+
+export interface UsdRepository {
+  getPriceDayUsd(tokenAddress: string, date: Date): Promise<number>;
+}
+
+export class UsdRepositoryMock {
```

Comment:
Nitpick: why doesn't the mock implement the interface? =P

Reply:
It should, is not a nit is a good catch :)